scripts/ocha/geodata/README.md

```diff
@@ -0,0 +1,57 @@
+# OCHA Geodata
+
+This directory contains scripts to turn shapefiles from UN OCHA
+(https://data.humdata.org) into MCF suitable for import into Data Commons.
+
+This includes geojsons for the following countries:
+* [Nepal](https://data.humdata.org/dataset/administrative-bounadries-of-nepal)
+* [Pakistan](https://data.humdata.org/dataset/pakistan-administrative-level-0-1-2-and-3-boundary-polygons-lines-and-central-places)
+* [Bangladesh](https://data.humdata.org/dataset/administrative-boundaries-of-bangladesh-as-of-2015#)
+* [China](https://data.humdata.org/dataset/cod-ab-chn)
+
+## Generation
+
+1. Download and prepare geojsons.  This depends on `ogr2ogr` tool.
```

Comment:
say more about "ogr2ogr", is this a library?

Reply:
Done.